openedx/features/course_duration_limits/models.py

```diff
@@ -78,7 +78,7 @@ def enabled_for_enrollment(cls, enrollment=None, user=None, course_key=None):
         # enrollment might be None if the user isn't enrolled. In that case,
         # return enablement as if the user enrolled today
         if enrollment is None:
-            return cls.enabled_for_course(course_key=course_key, target_date=datetime.utcnow().date())
+            return cls.enabled_for_course(course_key=course_key, target_datetime=timezone.now())
```

Comment:
out of curiosity what is the different between datetime.utcnow() and timezone.now()

Reply:
`datetime.utcnow()` returns a naive datetime. `timezone.now()` returns a timezone aware datetime that is UTC